### ISSUE TITLE
Filter out already downloaded images before starting download.

### ISSUE THREAD

**Issue by FourTOne5** [tachiyomiorg/tachiyomi#5966]
### Describe your suggested feature

Currently after pausing and resuming or rearranging the queue the downloader goes page by page of a chapter and checks if the page is downloaded or not.

When you have some network problem the downloader seems to pause and start now this isn't much of a issue for manga with 30 or more pages but when downloading stuff with pages in hundred and seeing the download start from 0 can be tedious.

So what I am asking is that before starting a download filter out already downloaded pages and try to download pages that have not been downloaded.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.12.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.
